packages/edit-site/src/components/header/more-menu/copy-content-menu-item.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { useCopyToClipboard } from '@wordpress/compose';
+import { store as noticesStore } from '@wordpress/notices';
+import { store as coreStore } from '@wordpress/core-data';
+import { __unstableSerializeAndClean } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../../store';
+
+export default function CopyContentMenuItem() {
+	const { createNotice } = useDispatch( noticesStore );
+	const getText = useSelect( ( select ) => {
+		return () => {
```

Comment:
This hook is a bit more complex than the one on the edit post screen:

https://github.com/WordPress/gutenberg/blob/22111f5bf2ed9cbb6db558ea7c03888c4f593b14/packages/edit-post/src/plugins/copy-content-menu-item/index.js#L13-L17

Reply:
Hi @gziolo that happens because the editor package has a special case to handle the content attribute. In that case getEditedPostContent selector is called.
That selector essentially does what I'm doing here:
https://github.com/WordPress/gutenberg/blob/d754759c45a4f882b01a78e1a2064b77e9d87370/packages/editor/src/store/selectors.js#L918

Reply:
So it’s there but hidden 👍

So we need to do some testing and it’s good to go.